src/__builtins.di

```diff
@@ -68,4 +68,14 @@ version (DigitalMars)
     alias __builtin_fabs  = core.stdc.math.fabs;
     alias __builtin_fabsf = core.stdc.math.fabsf;
     alias __builtin_fabsl = core.stdc.math.fabsl;
+
+    import core.bitop;
+
+    alias __builtin_bswap32 = core.bitop.bswap;
+    alias __builtin_bswap64 = core.bitop.bswap;
+
+    // Stub these out to no-ops
+    enum  __builtin_constant_p = 0; // should be something like __traits(compiles, enum X = expr)
+    long  __builtin_expect(long exp, long c) { return 0; }
```

Comment:
`return 0;` is not what `__builtin_expect` does. It should `return exp;`

Reply:
yah, you're right. I should go to bed now :-)